academic-services/knowledge-exploration-service/tutorial-grammar-design.md

```diff
@@ -0,0 +1,401 @@
+    ```
+
+    The output should mirror the following JSON:
+
+    ```json
+    {
+    "expr": "All()",
+    "entities": [
+      {
+        "logprob": -17.478,
+        "prob": 2.56685328E-08,
+        "Abstract": "In this paper we describe a new release of a Web scale entity graph that serves as the backbone of Microsoft Academic Service (MAS), a major production effort with a broadened scope to the namesake vertical search engine that has been publicly available since 2008 as a research prototype. At the core of MAS is a heterogeneous entity graph comprised of six types of entities that model the scholarly activities: field of study, author, institution, paper, venue, and event. In addition to obtaining these entities from the publisher feeds as in the previous effort, we in this version include data mining results from the Web index and an in-house knowledge base from Bing, a major commercial search engine. As a result of the Bing integration, the new MAS graph sees significant increase in size, with fresh information streaming in automatically following their discoveries by the search engine. In addition, the rich entity relations included in the knowledge base provide additional signals to disambiguate and enrich the entities within and beyond the academic domain. The number of papers indexed by MAS, for instance, has grown from low tens of millions to 83 million while maintaining an above 95% accuracy based on test data sets derived from academic activities at Microsoft Research. Based on the data set, we demonstrate two scenarios in this work: a knowledge driven, highly interactive dialog that seamlessly combines reactive search and proactive suggestion experience, and a proactive heterogeneous entity recommendation.",
+        "DOI": "10.1145/2740908.2742839",
+        "EstimatedCitationCount": "418",
+        "FullTextUrl": "http://localhost/example-full-text-link-2",
+        "Title": "An Overview of Microsoft Academic Service (MAS) and Applications",
+        "VenueFullName": "The Web Conference",
+        "Year": 2015,
+        "AuthorAffiliations": [
+          {
+            "NormalizedAffiliationName": "microsoft",
+            "NormalizedAuthorName": "arnab sinha",
+            "AuthorName": "Arnab Sinha",
+            "Sequence": "1"
+          },
+          {
+            "NormalizedAffiliationName": "microsoft",
+            "NormalizedAuthorName": "zhihong shen",
+            "AuthorName": "Zhihong Shen",
+            "Sequence": "2"
+          },
+          ...
+        ],
+        "FieldsOfStudy": [
+          {
+            "Name": "World Wide Web",
+            "NormalizedName": "world wide web"
+          },
+          {
+            "Name": "Vertical search",
+            "NormalizedName": "vertical search"
+          },
+          ...
+        ]
+      }
+    ],
+    "timed_out": false
+    }
+    ```
+
+1. Run Evaluate command using the built index to verify index operations are working as expected:
```

Comment:
>Run Evaluate command using the built index to verify index operations are working as expected: [](start = 3, length = 94)

Shouldn't you verify one of the new attributes? Otherwise this isn't technically validating that the schema was changed from the previous tutorial :-)